Ohana/Classes/Common/DataProviders/OHABAddressBookContactsDataProvider.h

```diff
@@ -49,6 +49,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (instancetype)init NS_UNAVAILABLE;
 
++ (OHABAddressBookContactsDataProvider *)new NS_UNAVAILABLE;
```

Comment:
Seems like a good idea, but should be done to all or none of the classes with unavailable inits.  Let's make this change a separate PR since it spans a large number of files (I see 12 from a quick search).  Also for consistency we should probably make it `+ (instancetype)new NS_UNAVAILABLE;`.